plugins/gradle/build.gradle

```diff
@@ -161,8 +166,27 @@ tasks.named('check') {
     dependsOn(tasks.named("integrationTests"))
 }
 
+shadowJar {
+    classifier = null
+}
+
 artifacts {
-    archives jar
+    archives shadowJar
+}
+
+gradlePlugin {
+    website = 'https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android/'
```

Comment:
Suspect that we need to add additional info here.
Sonatype close repo is failed due to error message:
Invalid POM: /com/newrelic/agent/android/gradle/7.6.0/gradle-7.6.0.pom: Project name missing, Project description missing, Project URL missing, License information missing, SCM URL missing, Developer information missing

Reply:
This block is only for teh Gradle Plugin Portal. The error is from Sonatype, but other POM changes to get GPP to work are probably the cause.